UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;
+  if (value == -1)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_exchange_rate_loading"
+        small
+        withOverlay={false}
+      />
+    );
+  return (
+    <div>
+      {loadingIcon}

```

Comment:
Probably you want to do something like this:
```js
{ !value && loadingIcon }
```
This will showcase the `loadingIcon` only when `value` is not assigned. We use [Short-circuit evaluation](https://en.wikipedia.org/wiki/Short-circuit_evaluation) to ensure that only it's show when the first condition is true.

Reply:
fixed